lib/dynflow/world.rb

```diff
@@ -52,7 +52,14 @@ def subscribed_actions(action_class)
 
     # reload actions classes, intended only for devel
     def reload!
-      @action_classes.map! { |klass| klass.to_s.constantize }
+      # TODO what happens with newly loaded classes
+      @action_classes.map! do |klass|
```

Comment:
Should not we remove the missing classes from the array, to not rescue from the load error every time?